packaging/rpm/microshift.spec

```diff
@@ -95,11 +94,8 @@ Things break. But you can still help shape it, too.
 
 %package selinux
 Summary: SELinux policies for MicroShift
-BuildRequires: selinux-policy >= %{selinux_policyver}
-BuildRequires: selinux-policy-devel >= %{selinux_policyver}
```

Comment:
Are we sure this works now on RHEL8? it was added because of that platform. We can try...

Reply:
```
[vagrant@rhel8 microshift]$ cat packaging/rpm/_rpmbuild/SPECS/microshift.spec | grep -A5 SELinux 
# SELinux specifics
%global selinuxtype targeted
%define container_policyver 2.167.0-1
%define container_policy_epoch 2
%define microshift_relabel_files() \
   mkdir -p /var/hpvolumes; \
--
Summary: SELinux policies for MicroShift
Requires: container-selinux >= %{container_policy_epoch}:%{container_policyver}
BuildArch: noarch

%description selinux
SElinux policy modules for MicroShift.
--
# SELinux modules build

cd packaging/selinux
make

%install
```

After `make rpm`

```
[vagrant@rhel8 microshift]$ sudo dnf install packaging/rpm/_rpmbuild/RPMS/noarch/microshift-selinux-4.8.0-2022_03_11_124751_6_gcf34c42a.el8.noarch.rpm packaging/rpm/_rpmbuild/RPMS/x86_64/microshift-4.8.0-202
2_03_11_124751_6_gcf34c42a.el8.x86_64.rpm                                                                                                                                                                                                               
Updating Subscription Management repositories.
Last metadata expiration check: 1:54:21 ago on Tue 29 Mar 2022 10:17:08 AM UTC.                                                                                                                                
Dependencies resolved.
===============================================================================================================================================================================================================                                         
 Package                                          Architecture                         Version                                                                Repository                                  Size                                          
===============================================================================================================================================================================================================                                         
Installing:        
 microshift                                       x86_64                               4.8.0-2022_03_11_124751_6_gcf34c42a.el8                                @commandline                                36 M                                          
 microshift-selinux                               noarch                               4.8.0-2022_03_11_124751_6_gcf34c42a.el8                                @commandline                                19 k                                          
                                                   
Transaction Summary     
===============================================================================================================================================================================================================                                         
Install  2 Packages
                                                                                                                                                                                                               
Total size: 36 M                                                                                                                                                                                               
Installed size: 154 M                                                                                                                                                                                          
Is this ok [y/N]: y                                                                                                                                                                                            
Downloading Packages:                                                                                                                                                                                          
Running transaction check                                                                                                                                                                                      
Transaction check succeeded.                                                                                                                                                                                   
Running transaction test                                                                                                                                                                                       
Transaction test succeeded.                                                                                                                                                                                    
Running transaction        
  Preparing        :                                                                                                                                                                                       1/1                                          
  Installing       : microshift-selinux-4.8.0-2022_03_11_124751_6_gcf34c42a.el8.noarch                                                                                                                     1/2                                          
  Running scriptlet: microshift-selinux-4.8.0-2022_03_11_124751_6_gcf34c42a.el8.noarch                                                                                                                     1/2 
  Installing       : microshift-4.8.0-2022_03_11_124751_6_gcf34c42a.el8.x86_64                                                                                                                             2/2                                          
  Running scriptlet: microshift-4.8.0-2022_03_11_124751_6_gcf34c42a.el8.x86_64                                                                                                                             2/2                                          
  Running scriptlet: microshift-selinux-4.8.0-2022_03_11_124751_6_gcf34c42a.el8.noarch                                                                                                                     2/2                                          
  Running scriptlet: microshift-4.8.0-2022_03_11_124751_6_gcf34c42a.el8.x86_64                                                                                                                             2/2                                          
  Verifying        : microshift-selinux-4.8.0-2022_03_11_124751_6_gcf34c42a.el8.noarch                                                                                                                     1/2                                          
  Verifying        : microshift-4.8.0-2022_03_11_124751_6_gcf34c42a.el8.x86_64                                                                                                                             2/2                                          
Installed products updated.                                                                            
                                                   
Installed:
  microshift-4.8.0-2022_03_11_124751_6_gcf34c42a.el8.x86_64                                          microshift-selinux-4.8.0-2022_03_11_124751_6_gcf34c42a.el8.noarch                                                                                  
                                                   
Complete!         
```

Starting the systemd service, and after a while:

```
[vagrant@rhel8 microshift]$ oc get pods -A
NAMESPACE                       NAME                                  READY   STATUS    RESTARTS   AGE
kube-system                     kube-flannel-ds-68pwq                 1/1     Running   0          2m30s
kubevirt-hostpath-provisioner   kubevirt-hostpath-provisioner-7sxkt   1/1     Running   0          2m10s
openshift-dns                   dns-default-gfz2t                     2/2     Running   0          2m30s
openshift-dns                   node-resolver-vs55f                   1/1     Running   0          2m30s
openshift-ingress               router-default-6c96f6bc66-6t47v       1/1     Running   0          2m34s
openshift-service-ca            service-ca-7bffb6f6bf-vlrks           1/1     Running   0          2m35s
```

```
 cat /etc/redhat-release 
Red Hat Enterprise Linux release 8.4 (Ootpa)
```